docs/src/main/tut/add-support-for-simple-types.md

```diff
@@ -0,0 +1,61 @@
+### Add support for simple types
+
+Let's see how to support the type
```

Comment:
Do you think there should be any exposition here for why the type isn't supported? My concern is that someone might land here expecting they need to follow these steps in order to get a `case class` working. 

Perhaps something like: Out of the box, PureConfig supports `case class`es, but not regular classes. Let's see how to support a simple class with a single mutable field.